projects/ngrx.io/src/app/custom-elements/events/event.service.spec.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  HttpClientTestingModule,
+  HttpTestingController,
+} from '@angular/common/http/testing';
+import { Injector } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { EventService } from './event.service';
+import { Event, EventResponse } from './event.model';
+
+describe('EventService', () => {
+  let injector: Injector;
+  let eventService: EventService;
+  let httpMock: HttpTestingController;
+
+  beforeEach(() => {
+    injector = TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
+      providers: [EventService],
+    });
+
+    eventService = injector.get<EventService>(EventService);
+    eventService.currentDate = new Date('01-02-2019');
+    httpMock = injector.get(HttpTestingController);
+  });
+
+  afterEach(() => httpMock.verify());
+
+  it('should make a single connection to the server', () => {
+    const req = httpMock.expectOne({});
+    expect(req.request.url).toBe('generated/events.json');
+  });
+
+  describe('#events', () => {
```

Comment:
Just wondering. Why `#events`?

Reply:
Just following `resource.service.spec.ts`.  Fixed in 387753c191e9ae2120f12e5598c8a15715c47801.